CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 Go-Common is the library of common functions for Tidepool's Go-based applications
 
+## 2.1.0 - 2024-11-12
+### Added
+- Util lib to build http requests (requestBuilder)
+
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**🤓 nitpick:** 
enhance the changelog with something like : 
- changed new functions for session handlers
- removed : portal client